.travis.yml

```diff
@@ -30,22 +28,27 @@ addons:
     packages:
       - opencl-headers
       - lcov
-
+      - python-pip
 cache:
     directories:
         - ${HOME}/${BOOST_BASENAME}/boost
         - ${HOME}/${BOOST_BASENAME}/stage/lib
         - ${AMDAPPSDKROOT}
 
 before_install:
+    - if [ "$VEXCL_BACKEND" = "JIT" ] && [ "$CC" = "clang" ] ; then pip install --user cmake ; fi
```

Comment:
Do we still need to install the latest cmake (and pip in line 31)? Otherwise, this looks ready to merge.

Reply:
No, but it might be nice to have a test with the latest CMake too. No reason for it to be this one, though, so I'll try removing this.